sdk/eventhub/azure-eventhub/azure/eventhub/_consumer_client.py

```diff
@@ -72,6 +72,15 @@ class EventHubConsumerClient(ClientBase):
      The failed internal partition consumer will be closed (`on_partition_close` will be called if provided) and
      new internal partition consumer will be created (`on_partition_initialize` will be called if provided) to resume
      receiving.
+    :keyword float retry_backoff_factor: A backoff factor to apply between attempts after the second try
+     (most errors are resolved immediately by a second try without a delay).
+     In fixed mode, retry policy will always sleep for {backoff factor}.
+     In 'exponential' mode, retry policy will sleep for: `{backoff factor} * (2 ** ({number of total retries} - 1))`
+     seconds. If the backoff_factor is 0.1, then the retry will sleep
+     for [0.0s, 0.2s, 0.4s, ...] between retries. The default value is 0.8.
+    :keyword int retry_backoff_max: The maximum back off time. Default value is 120 seconds (2 minutes).
```

Comment:
Python supports float-type timeout as well.
```suggestion
    :keyword float retry_backoff_max: The maximum back off time. Default value is 120 seconds (2 minutes).
```